platform/barefoot/sonic-platform-modules-bfn-montara/sonic_platform/thermal.py

```diff
@@ -72,23 +72,48 @@ def __init__(self, chip, label):
         self.__chip = chip
         self.__label = label
         self.__name = f"{chip}:{label}".lower().replace(' ', '-')
+        self.collect_temp = []
```

Comment:
looks like that a code use encapsulation and separates private (start with 2 underscores `self.__private_var`) and public (`self.public_var`) implementations.
Are you sure that this member variable must be public?